setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='0.1.0',
+    version='1.0',
```

Comment:
This seems like quite a jump in the version number, how did we decide to label it 1.0?